frontend/src/components/Tokenomics/PaydayPoolReward.vue

```diff
@@ -70,5 +72,12 @@ type Props = {
 	goToPage: (page: PageInfo) => (target: PaginationTarget) => void
 }
 
-defineProps<Props>()
+const props = defineProps<Props>()
+
+const showFinalization = computed(() => {
```

Comment:
This seems like a duplicate of what you have in `PaydayAccountReward`. Could those classes share more code?